src/SpeedyWeather.jl

```diff
@@ -24,6 +27,11 @@ import JLD2: jldopen, jldsave, JLDFile
 import CodecZlib
 import BitInformation: round, round!
 import ProgressMeter
+import ModelParameters
+import ConstructionBase: constructorof, getproperties, setproperties
+
+# DOMAINS
+using DomainSets
```

Comment:
@milankl I know you prefer `import` generally but I think `using` is easier for `DomainSets`. Also, VS Code incorrectly highlights `import DomainSets: :(..)` (the interval operator) as a syntax error which is kind of annoying...

Reply:
Fine with me